src/samuel/feature.js

```diff
@@ -11,3 +11,59 @@ null - unidentified
 "pi" - string
 67 + 18 - number
 */
+
+//CONVERTING BINARY TO DECIMAL
+function DecimalToBinary(DecimalValue){
+		var BinaryValue = '';
+		// Loop from 2^64/2 to 1
+		for (var i=64; i>=1; i--){
```

Comment:
You need to surround your operators with spaces to make your code easier to read.